### PR TITLE
Change  two check boxes in CleanupPresetPanel.fxml to radio buttons with Toggle Group

### LINKED ISSUE #10583: Change checkbox to a radio button in Cleanup for converting to biblatex/bibatex
**Is your suggestion for improvement related to a problem? Please describe.**
Currently, two checkboxes are displayed for "Convert to BibTeX" and "Convert to biblatex" but one can only choose one. 

**Describe the solution you'd like**
The checkboxes should be replaced by radio buttons with a Toggle Group

**Additional context**

grafik

### PR DESCRIPTION
**Pull request by suxiaocha123**
Resolves #10583

### Mandatory checks

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by ThiloteE** [comment]
To ease organizational workflows I have linked the pull-request to the issue with syntax as described in https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

> <h2 id="linking-a-pull-request-to-an-issue-using-a-keyword">Linking a pull request to an issue using a keyword</h2>
> <p>You can link a pull request to an issue by using a supported keyword 
> in the pull request's description or in a commit message. The pull 
> request <strong>must be</strong> on the default branch.</p>
> <ul>
> <li>close</li>
> <li>closes</li>
> <li>closed</li>
> <li>fix</li>
> <li>fixes</li>
> <li>fixed</li>
> <li>resolve</li>
> <li>resolves</li>
> <li>resolved</li>
> </ul>
> <p>If you use a keyword to reference a pull request comment in another 
> pull request, the pull requests will be linked. Merging the referencing 
> pull request also closes the referenced pull request.</p>
> <p>The syntax for closing keywords depends on whether the issue is in the same repository as the pull request.</p>

File: src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.fxml [hunk 1]
```diff
@@ -30,8 +30,13 @@
           <Insets left="20.0" />
        </VBox.margin>
       </VBox>
-      <CheckBox fx:id="cleanUpBiblatex" text="%Convert to biblatex format (e.g., store publication date in date field)" />
-      <CheckBox fx:id="cleanUpBibtex" text="%Convert to BibTeX format (e.g., store publication date in year and month fields)" />
+     <RadioButton text="%Convert to biblatex format (e.g., store publication date in date field)">
+      <toggleGroup>
+        <ToggleGroup fx:id="conversionToggleGroup" />
+       </toggleGroup>
+   </RadioButton>
```
Comment (koppor) [inline src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.fxml:37]: Indent seems tob e off. Opening and matching closing tags should be on the same column.

File: src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.fxml [hunk 2]
```diff
@@ -30,8 +30,13 @@
...
+
+   <RadioButton text="%Convert to BibTeX format (e.g., store publication date in year and month fields)" toggleGroup="$conversionToggleGroup" />
```
Comment (koppor) [inline src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.fxml:39]: Indent is off here, too.

**Review by koppor — CHANGES_REQUESTED** [review]
CHANGELOG.md entry is missing.

I get an exception when trying out

image

**Comment by koppor** [comment]
Follow-up at https://github.com/JabRef/jabref/pull/10601.

**Pull request closed without merging**